src/components/ec-timer/ec-timer.vue

```diff
@@ -0,0 +1,131 @@
+<template>
+  <div
+    class="ec-timer"
+    data-test="ec-timer"
+  >
+    <svg
+      :width="diameter"
+      :height="diameter"
+      :viewBox="viewbox"
+      class="ec-timer__svg"
+    >
+      <circle
+        class="ec-timer__background"
+        cx="50%"
+        cy="50%"
+        :r="radius"
+        fill="transparent"
+        :stroke-width="strokeWidth"
+      />
+
+      <circle
+        class="ec-timer__remaining"
+        cx="50%"
+        cy="50%"
+        :r="radius"
+        fill="transparent"
+        :stroke-width="strokeWidth"
+        :stroke-dasharray="circumference"
+        :stroke-dashoffset="offset"
+      />
+
+      <text
+        x="50%"
+        y="50%"
+        class="ec-timer__text"
+        dominant-baseline="central"
+        text-anchor="middle"
+      >
+        {{ secondsLeft }}s
```

Comment:
I think this text should be translated. Not sure how this would be displayed in Chinese, but I'm pretty sure, Russian language doesn't have `20s` but `20c`.

Reply:
Ha, thought was universal the "s". prop then I guess

Reply:
After talk with the team, will be slot